--- ncurses-6.5-20250201/misc/terminfo.src~	2025-01-04 22:02:11.000000000 +0100
+++ ncurses-6.5-20250201/misc/terminfo.src	2025-02-08 10:47:29.309930740 +0100
@@ -4895,7 +4895,7 @@ xterm-r6|xterm X11R6 version,
 	use=ansi+erase, use=ansi+idl, use=ansi+local,
 	use=ansi+sgrso, use=ansi+sgrul, use=vt220+vtedit,
 	use=xterm+kbs, use=vt100+enq, use=vt220+sfkeys,
-	use=xterm+r5+fkeys,
+	use=xterm+pcfkeys,
 
 xterm-old|antique xterm version,
 	use=xterm-r6,
