--- ncurses-5.4/misc/terminfo.src.orig	Fri Jun 18 14:46:01 2004
+++ ncurses-5.4/misc/terminfo.src	Fri Jun 18 14:47:06 2004
@@ -3663,7 +3663,7 @@
 	sgr=\E[0%?%p6%t;1%;%?%p1%t;3%;%?%p2%t;4%;%?%p3%t;7%;%?%p4%t;5%;m%?%p9%t\016%e\017%;, 
 	sgr0=\E[m, smacs=^N, smcup=\E[?1049h, smir=\E[4h, 
 	smkx=\E[?1h\E=, smso=\E[3m, smul=\E[4m, tbc=\E[3g, E0=\E(B, 
-	S0=\E(%p1%c, use=ecma+color, 
+	S0=\E(%p1%c, use=ecma+color, hpa=\E[%i%p1%dG,
 # The bce and status-line entries are from screen 3.9.13 (and require some
 # changes to .screenrc).
 screen-bce|VT 100/ANSI X3.64 virtual terminal with bce, 
