--- ncurses-5.6/mk-1st.awk.orig	2006-12-30 17:40:34.164755215 +0100
+++ ncurses-5.6/mk-1st.awk	2006-12-30 17:42:11.054276635 +0100
@@ -83,7 +83,7 @@
 		}
 	}
 function make_shlib(objs, shlib_list) {
-		printf "\t$(MK_SHARED_LIB) $(%s_OBJS) $(%s) $(LDFLAGS)\n", objs, shlib_list
+		printf "\t$(MK_SHARED_LIB) $(LDFLAGS) $(%s_OBJS) $(%s)\n", objs, shlib_list
 	}
 function sharedlinks(directory) {
 		if ( ShlibVer != "auto" && ShlibVer != "cygdll" ) {
diff -Nur ncurses-5.5.orig/progs/Makefile.in ncurses-5.5.linking/progs/Makefile.in
--- ncurses-5.5.orig/progs/Makefile.in	2006-08-05 18:50:12.000000000 +0200
+++ ncurses-5.5.linking/progs/Makefile.in	2006-08-05 18:50:21.000000000 +0200
@@ -96,7 +96,8 @@
 LD		= @LD@
 LINK		= @LINK_PROGS@ $(LIBTOOL_LINK) $(CC)
 LDFLAGS		= @EXTRA_LDFLAGS@ \
-		@TINFO_ARGS@ @LDFLAGS@ @LD_MODEL@ @LIBS@
+		@LDFLAGS@ @LD_MODEL@
+LIBS		= @TINFO_ARGS@ @LIBS@
 
 LDFLAGS_LIBTOOL	= $(LDFLAGS) $(CFLAGS_LIBTOOL)
 LDFLAGS_NORMAL	= $(LDFLAGS) $(CFLAGS_NORMAL)
@@ -207,40 +208,40 @@
 	$(MODEL)/dump_entry$o
 
 tic$x: $(DEPS_TIC) $(DEPS_CURSES) transform.h
-	@ECHO_LINK@ $(LINK) $(DEPS_TIC) $(LDFLAGS_DEFAULT) -o $@
+	@ECHO_LINK@ $(LINK) $(LDFLAGS_DEFAULT) $(DEPS_TIC) $(LIBS) -o $@
 
 DEPS_TOE = \
 	$(MODEL)/toe$o \
 	$(MODEL)/dump_entry$o
 
 toe$x: $(DEPS_TOE) $(DEPS_CURSES)
-	@ECHO_LINK@ $(LINK) $(DEPS_TOE) $(LDFLAGS_DEFAULT) -o $@
+	@ECHO_LINK@ $(LINK) $(LDFLAGS_DEFAULT) $(DEPS_TOE) $(LIBS) -o $@
 
 DEPS_CLEAR = \
 	$(MODEL)/clear$o
 
 clear$x: $(DEPS_CLEAR) $(DEPS_CURSES)
-	@ECHO_LINK@ $(LINK) $(DEPS_CLEAR) $(LDFLAGS_DEFAULT) -o $@
+	@ECHO_LINK@ $(LINK) $(LDFLAGS_DEFAULT) $(DEPS_CLEAR) $(LIBS) -o $@
 
 DEPS_TPUT = \
 	$(MODEL)/tput$o
 
 tput$x: $(DEPS_TPUT) $(DEPS_CURSES) transform.h
-	@ECHO_LINK@ $(LINK) $(DEPS_TPUT) $(LDFLAGS_DEFAULT) -o $@
+	@ECHO_LINK@ $(LINK) $(LDFLAGS_DEFAULT) $(DEPS_TPUT) $(LIBS) -o $@
 
 DEPS_INFOCMP = \
 	$(MODEL)/infocmp$o \
 	$(MODEL)/dump_entry$o
 
 infocmp$x: $(DEPS_INFOCMP) $(DEPS_CURSES)
-	@ECHO_LINK@ $(LINK) $(DEPS_INFOCMP) $(LDFLAGS_DEFAULT) -o $@
+	@ECHO_LINK@ $(LINK) $(LDFLAGS_DEFAULT) $(DEPS_INFOCMP) $(LIBS) -o $@
 
 DEPS_TSET = \
 	$(MODEL)/tset$o \
 	$(MODEL)/dump_entry$o
 
 tset$x: $(DEPS_TSET) $(DEPS_CURSES) transform.h
-	@ECHO_LINK@ $(LINK) $(DEPS_TSET) $(LDFLAGS_DEFAULT) -o $@
+	@ECHO_LINK@ $(LINK) $(LDFLAGS_DEFAULT) $(DEPS_TSET) $(LIBS) -o $@
 
 termsort.c: $(srcdir)/MKtermsort.sh
 	sh $(srcdir)/MKtermsort.sh $(AWK) $(srcdir)/../include/@TERMINFO_CAPS@ >$@
diff -Nur ncurses-5.5.orig/tack/Makefile.in ncurses-5.5.linking/tack/Makefile.in
--- ncurses-5.5.orig/tack/Makefile.in	2005-06-19 01:01:59.000000000 +0200
+++ ncurses-5.5.linking/tack/Makefile.in	2006-08-05 18:51:31.000000000 +0200
@@ -64,7 +64,8 @@
 LD		= @LD@
 LINK		= @LINK_PROGS@ $(LIBTOOL_LINK) $(CC)
 LDFLAGS		= @EXTRA_LDFLAGS@ \
-		@TINFO_ARGS@ @LDFLAGS@ @LD_MODEL@ @LIBS@
+		@LDFLAGS@ @LD_MODEL@
+LIBS		= @TINFO_ARGS@ @LIBS@
 
 LDFLAGS_LIBTOOL	= $(LDFLAGS) $(CFLAGS_LIBTOOL)
 LDFLAGS_NORMAL	= $(LDFLAGS) $(CFLAGS_NORMAL)
@@ -135,7 +136,7 @@
 	$(MODEL)/tack$o
 
 tack$x: $(DEPS_TACK) $(DEPS_CURSES)
-	@ECHO_LINK@ $(LINK) $(DEPS_TACK) $(LDFLAGS_DEFAULT) -o $@
+	@ECHO_LINK@ $(LINK) $(LDFLAGS_DEFAULT) $(DEPS_TACK) $(LIBS) -o $@
 
 #
 # Utility productions start here
