--- ncurses-5.2/misc/terminfo.src~	Wed Dec 26 11:13:07 2001
+++ ncurses-5.2/misc/terminfo.src	Wed Dec 26 11:17:24 2001
@@ -2441,7 +2441,7 @@
 # "resize -s"), though it does not pass SIGWINCH to the application if the
 # user resizes the window with the mouse.
 teraterm|Tera Term Pro, 
-	km, xon@, 
+	xon@, 
 	ncv#43, vt@, 
 	acsc=+\020\,\021-\030.^Y0\333`\004a\261f\370g\361h\260i\316j\331k\277l\332m\300n\305o~p\304q\304r\304s_t\303u\264v\301w\302x\263y\363z\362{\343|\330}\234~\376, 
 	blink=\E[5m, bold=\E[1m, civis=\E[?25l, clear=\E[H\E[J, 
@@ -2533,7 +2533,7 @@
 # (khome/kend do not actually work in X11R5 or X11R6, but many people use this
 # for compatibility with other emulators).
 xterm-r6|xterm-old|xterm X11R6 version, 
-	OTbs, am, km, mir, msgr, xenl, 
+	OTbs, am, mir, msgr, xenl, 
 	cols#80, it#8, lines#24, 
 	acsc=``aaffggiijjkkllmmnnooppqqrrssttuuvvwwxxyyzz{{||}}~~, 
 	bel=^G, bold=\E[1m, clear=\E[H\E[2J, cr=^M, 
@@ -2938,7 +2938,7 @@
 # Since rxvt is not really compatible with xterm, it should be configured as
 # "rxvt" (monochrome) and "rxvt-color". 
 rxvt-basic|rxvt terminal base (X Window System), 
-	OTbs, am, bce, eo, km, mir, msgr, xenl, xon, 
+	OTbs, am, bce, eo, mir, msgr, xenl, xon, 
 	cols#80, it#8, lines#24, 
 	acsc=``aaffggjjkkllmmnnooppqqrrssttuuvvwwxxyyzz{{||}}~~, 
 	bel=^G, blink=\E[5m, bold=\E[1m, civis=\E[?25l, 
@@ -2977,7 +2977,7 @@
 # removed kf0 which conflicts with kf10 -TD
 # remove cvvis which conflicts with cnorm -TD
 Eterm|Eterm-color|Eterm with xterm-style color support (X Window System), 
-	am, bce, bw, eo, km, mc5i, mir, msgr, xenl, xon, 
+	am, bce, bw, eo, mc5i, mir, msgr, xenl, xon, 
 	btns#5, cols#80, it#8, lines#24, lm#0, ncv@, 
 	acsc=``aaffggjjkkllmmnnooppqqrrssttuuvvwwxxyyzz{{||}}~~, 
 	bel=^G, blink=\E[5m, bold=\E[1m, civis=\E[?25l, 
@@ -3217,7 +3217,7 @@
 #
 # tested with screen 3.09.08
 screen|VT 100/ANSI X3.64 virtual terminal, 
-	OTbs, OTpt, am, km, mir, msgr, xenl, G0, 
+	OTbs, OTpt, am, mir, msgr, xenl, G0, 
 	colors#8, cols#80, it#8, lines#24, pairs#64, 
 	acsc=++\,\,--..00``aaffgghhiijjkkllmmnnooppqqrrssttuuvvwwxxyyzz{{||}}~~, 
 	bel=^G, blink=\E[5m, bold=\E[1m, cbt=\E[Z, civis=\E[?25l, 
