--- ncurses-5.6/misc/terminfo.src.orig	2006-10-07 19:52:03.000000000 +0200
+++ ncurses-5.6/misc/terminfo.src	2006-12-30 16:47:00.309608049 +0100
@@ -4161,7 +4161,7 @@
 	sgr=\E[0%?%p6%t;1%;%?%p1%t;3%;%?%p2%t;4%;%?%p3%t;7%;%?%p4%t;5%;m%?%p9%t\016%e\017%;,
 	sgr0=\E[m\017, smacs=^N, smcup=\E[?1049h, smir=\E[4h,
 	smkx=\E[?1h\E=, smso=\E[3m, smul=\E[4m, tbc=\E[3g, E0=\E(B,
-	S0=\E(%p1%c, use=ecma+color,
+	S0=\E(%p1%c, use=ecma+color, hpa=\E[%i%p1%dG,
 # The bce and status-line entries are from screen 3.9.13 (and require some
 # changes to .screenrc).
 screen-bce|VT 100/ANSI X3.64 virtual terminal with bce,
