--- ncurses-5.2/configure.orig	Mon Aug 12 09:10:54 2002
+++ ncurses-5.2/configure	Mon Aug 12 09:14:37 2002
@@ -10564,9 +10564,9 @@
 done
 echo "$as_me:10565: result: $cf_cv_src_modules" >&5
 echo "${ECHO_T}$cf_cv_src_modules" >&6
-TEST_ARGS="-L${LIB_DIR} $TEST_ARGS"
+TEST_ARGS="-L${LIB_DIR} -L../lib $TEST_ARGS"
 
-PROG_ARGS="-L${LIB_DIR} $PROG_ARGS"
+PROG_ARGS="-L${LIB_DIR} -L../lib $PROG_ARGS"
 
 SRC_SUBDIRS="man include"
 for cf_dir in $cf_cv_src_modules
