--- ncurses-5.9/Ada95/samples/Makefile.in~	2011-03-29 01:45:08.000000000 +0200
+++ ncurses-5.9/Ada95/samples/Makefile.in	2011-06-25 15:05:18.869032587 +0200
@@ -99,7 +99,7 @@
 CARGS        =-cargs $(ADAFLAGS)
 LARGS        =-largs -L../lib -lAdaCurses @TEST_ARG2@ $(LD_FLAGS) @TEST_LIBS2@
 
-PROGS        = tour$x ncurses$x @USE_GNAT_SIGINT@ rain$x 
+PROGS        = tour$x ncurses$x @USE_GNAT_SIGINT@
 
 all ::  $(PROGS)
 	@echo made $@
